src/main/java/io/temporal/internal/replay/DecisionsHelper.java

```diff
@@ -190,14 +189,15 @@ boolean handleActivityTaskCanceled(HistoryEvent event) {
   }
 
   boolean handleRequestCancelActivityTaskFailed(HistoryEvent event) {
-    RequestCancelActivityTaskFailedEventAttributes attributes =
-        event.getRequestCancelActivityTaskFailedEventAttributes();
-    String activityId = attributes.getActivityId();
-    long scheduledEventId = getActivityScheduledEventId(activityId);
-    DecisionStateMachine decision =
-        getDecision(new DecisionId(DecisionTarget.ACTIVITY, scheduledEventId));
-    decision.handleCancellationFailureEvent(event);
-    return decision.isDone();
+    //    RequestCancelActivityTaskFailedEventAttributes attributes =
```

Comment:
Looks like RequestCancelActivityTaskFailed is deprecated, but event type still exists.